test/testUtils/usdUtils.py

```diff
@@ -31,6 +33,20 @@
 
 usdSeparator = '/'
 
+def createStageFromFile(filePath: str):
+    """
+        Creates a USD stage from a given path to a USD file.
+        Args:
+            filePath (str): The file path to the USD file to load.
+        Returns :
+            A string representing the UFE path to the stage's proxy shape node
+    """
+    # Source the corresponding mel file, otherwise the procedure isn't available
+    mel.eval('source \"mayaUsd_createStageFromFile.mel\"')
+    loadStageCmd = f'mayaUsd_createStageFromFilePath(\"{filePath}\")'
+    # Replace backslashes with forward slashes to avoid them being interpreted
+    loadStageCmd = loadStageCmd.replace('\\', '/')
```

Comment:
This isn't great but it seems like the simplest way to get it to work, this is also what is done in MayaUSD ([example](https://github.com/Autodesk/maya-usd/blob/dev/test/lib/testMayaUsdCreateStageCommands.py#L72-L74)).